sources/packages/backend/apps/db-migrations/src/sql/CASSuppliers/Add-errors-column.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.cas_suppliers
+ADD
+    COLUMN errors VARCHAR(300) [];
```

Comment:
The business has a list of all possible errors. Since this is not under our control I would suggest checking the list or increasing the number.

Reply:
As part of the error pdf, the max length error is 110 characters, so i think we are safe.